csi.proto

```diff
@@ -1257,6 +1300,35 @@ message VolumeUsage {
   // Units by which values are measured. This field is REQUIRED.
   Unit unit = 4;
 }
+
+message VolumeHealth {
+  enum Condition {
+    // Volume health condition is unknown; treat it as healthy.
+    UNKNOWN = 0;
+    // Volume is healthy.
+    HEALTHY = 1;
+    // Volume is temporarily unhealthy, but will likely become healthy.
+    TEMPORARY_UNHEALTHY = 2;
+    // Volume will fail soon. Move your data off and replace volume or
```

Comment:
Volume is accessible currently, but a problem is detected that could lead to eventual data loss. Examples: recoverable hardware failure, cooling failure, equipment significantly past end of life.

Reply:
Done